cetmix_tower_server/demo/demo_data.xml

```diff
@@ -243,11 +251,12 @@
         <field name="name">Execute Python Code: Check Branch</field>
         <field name="action">python_code</field>
         <field name="code">
-if {{ branch }}:
-    COMMAND_RESULT={"exit_code": 0, "message": "Branch is defined!"}
-else:
-    COMMAND_RESULT={"exit_code": -1, "message": "Branch is not defined!"}
+            if {{ branch }}:
+            COMMAND_RESULT={"exit_code": 0, "message": "Branch is defined!"}
+            else:
+            COMMAND_RESULT={"exit_code": -1, "message": "Branch is not defined!"}
```

Comment:
why are these changes necessary here? also your changes is not python format 
please do not change the code formatting unless it is a bug, you are making an unnecessary update to git history.

Reply:
I think it's some formatting done by pre-commit, I'll fix it.